File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hashMap = {}
        for num in nums:
            if num in hashMap:
                hashMap[num] += 1
            else:
                hashMap[num] = 1
        for key, value in hashMap.items():
            if value == max(hashMap.values()):
                return key

File: Data Structures & Algorithms/majority-element-ii/submission-1.java
class Solution {
    public List<Integer> majorityElement(int[] nums) {      
        Map<Integer, Integer> myDict = new HashMap<>();
        List<Integer> myList = new ArrayList<>();
        int myLength = nums.length;
        for (int i = 0; i < nums.length; i++) {
            Integer count = myDict.get(nums[i]);
            if (count != null) {
                myDict.put(nums[i], count + 1);
            } else {
                myDict.put(nums[i], 1);
            }
        }
        for (Integer num : myDict.keySet()) {
            if (myDict.get(num) > myLength / 3) {
                myList.add(num);
            }
        } return myList;
    }
}